src/lib/components/Map/Map.js

```diff
@@ -97,17 +98,29 @@ class Map extends Component {
       this.map.on('zoom', this.handleLabelsOnMapZoom.bind(this))
 
       // Dispach map rendered to indicate map was rendered
-      this.props.dispatch(Actions.mapRendered(mapId));
+      this.props.dispatch(Actions.mapRendered(mapId, true));
     }
   }
 
   addMouseEvents(mapId) {
+    const { handlers } = this.props;
+    if (handlers && Array.isArray(handlers)) {
+      // let handler;
+
+      for (let c = 0; c < handlers.length; c += 1) {
+        this.map.on(handlers[c].type, handlers[c].method);
+      }
+    }
+
```

Comment:
@KipSigei - I wonder if we can add an optional prop(s) to the site-config to prevent the default event listeners in the case that a client has custom click events / mousemove events. We could add a check here for `APP.disableDefaultMapListeners` and return out of the function if true.

Then we could even take it a step further and wrap tooltip and onFeatureClick handlers in conditionals checking for something like `if (!APP.disableTooltip) { ...do tooltip stuff... }` and `if (!APP.disableDefaultFeatureClick) { ...do onFeatureClick stuff... }`.

What do you think? Not trying to further bloat the map-spec, but this could give clients fine-tune control over map events and could prevent conflicting events in the future.

cc @kahummer